.circleci/config.yml

```diff
@@ -105,11 +105,7 @@ references:
       environment:
         PACKAGES_DIR: *packages_workspace
       command: |
-        if [ "$(uname -s)" == "Darwin" ]; then
-            PKG_SUFFIX="osx-$(sw_vers -productVersion)"
-        elif [ "$(uname -s)" == "Linux" ]; then
-            PKG_SUFFIX="ubuntu"
```

Comment:
I am concerned this may open door to ambiguity of non-Ubuntu Linux support.

Though I see no easy way of addressing that - unless going back to if-elif:
```
epoch@ae-integration-epoch-n2:~$ uname -m
x86_64
epoch@ae-integration-epoch-n2:~$ uname -n
ae-integration-epoch-n2
epoch@ae-integration-epoch-n2:~$ uname -r
4.4.0-21-generic
epoch@ae-integration-epoch-n2:~$ uname -s
Linux
epoch@ae-integration-epoch-n2:~$ uname -v
#37-Ubuntu SMP Mon Apr 18 18:33:37 UTC 2016
```